pkg/sysctl/sysctl.go

```diff
@@ -30,6 +30,12 @@ func Validate(strSlice []string) (map[string]string, error) {
 		if len(arr) < 2 {
 			return nil, errors.Errorf("%s is invalid, sysctl values must be in the form of KEY=VALUE", val)
 		}
+
+		trimmed := fmt.Sprintf("%s=%s", strings.TrimSpace(arr[0]), strings.TrimSpace(arr[1]))
+		if trimmed != val {
+			return nil, errors.Errorf("'%s' is invalid, extra spaces found", val)
```

Comment:
```suggestion
			return nil, errors.Errorf("%q is invalid, extra spaces found", val)
```
`%q` takes care of quoting directly, same for the one below